### PR TITLE
Require tifffile with fix for area detector TIFFs.

### PR DESCRIPTION
**Pull request by danielballan**
Closes #9.

### CODE REVIEW

**Comment by danielballan** [comment]
- [x] build number is 0
- [x] dependencies updated

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ requirements:
     - python
     - dask
     - h5py
-    - tifffile
+    - tifffile >=2020.08.25
```
Comment (mrakitin) [inline recipe/meta.yaml]: The package's versioning scheme is not zero-padded, unfortunately. See https://github.com/nsls-ii-forge/tifffile-feedstock/blob/73194f5a61730a46c82ef3f356625d94ab1cae62/recipe/meta.yaml#L2.
```suggestion
    - tifffile >=2020.8.25
```

**Comment by danielballan** [comment]
Ah, good catch, @mrakitin. I double-checked that and somehow _still_ got it wrong. ("Should I copy-paste it to be sure?" I thought to myself. "Nah, I'll just type it." Oops.)

**Comment by mrakitin** [comment]
For some reason the PR merge (https://github.com/nsls-ii-forge/tifffile-feedstock/commit/73194f5a61730a46c82ef3f356625d94ab1cae62) failed to trigger a CI build, I've restarted the build manually.

**Comment by mrakitin** [comment]
By the way, I set a number of scheduled auto-tick-bot runs (every 2 hours during the business time), so that this kind of manual update may not be needed, we just need to wait a bit. I can set more frequent runs if needed.

See the duplicate PR #9.

**Review by mrakitin — APPROVED** [review]

**Comment by danielballan** [comment]
Oh, great! I wondered about that but since I didn't see a PR I just went ahead manually. I'll keep that in mind next time.

**Pull request merged**